pvlib/tests/iotools/test_tmy.py

```diff
@@ -29,6 +30,16 @@ def test_read_tmy3_norecolumn():
     assert 'GHI source' in data.columns
 
 
+def test_read_tmy3_map_variables():
+    data, meta = tmy.read_tmy3(TMY3_TESTFILE, map_variables=True)
+    assert 'ghi' in data.columns
```

Comment:
I would test a few more column mappings. Acutally, why not just test all of the mapped columns for good measure.

Reply:
agree